changelog.md

```diff
@@ -9,11 +9,12 @@
 * `CHG` Improve performance of multithreaded `--check` and `undefined-field` diagnostic
 * `FIX` Addons can now self-recommend as expected. Fixed by correcting the `wholeMatch` function
 * `FIX` Now correctly evaluates the visibility of fields in a class when they are defined directly in the object. use for completion and invisible dianostic. [#2752](https://github.com/LuaLS/lua-language-server/issues/2752)
-* `NEW` added lua regular expression support for Lua.doc.<scope>Name [#2753](https://github.com/LuaLS/lua-language-server/pull/2753)
+* `NEW` Added lua regular expression support for Lua.doc.\<scope\>Name [#2753](https://github.com/LuaLS/lua-language-server/pull/2753)
```

Comment:
It would be better to use backtick for `` `code related` `` in markdown
=> `` `Lua.doc.<scope>Name` ``
```suggestion
* `NEW` Added lua regular expression support for `Lua.doc.<scope>Name` [#2753](https://github.com/LuaLS/lua-language-server/pull/2753)
```

In this way no need to escape special characters, they are rendered as is.

---

Below shows the differences between the two:
* `NEW` Added lua regular expression support for Lua.doc.\<scope\>Name
* `NEW` Added lua regular expression support for `Lua.doc.<scope>Name`